Automattic/gutenberg-everywhere
# Add/search in content embed

Adds the ability to search for WP.com support pages directly from the Content Embed block.

## Testing instructions

- Deploy this PR to sandbox
- In the editor, create a new Content Block by pressing `/`
- In the input field, enter some keywords, for example, 'Change Domain'
- Verify you are presented with the list of the relevant support pages
- Click on the support page item to insert its URL.
